### ISSUE TITLE
Pitch calculation error

### ISSUE THREAD

**Issue by denjhang** [digital-sound-antiques/vgm-conv#42]
![截图未命名0817193238](https://user-images.githubusercontent.com/61811583/129719016-bb277a12-9f16-4797-92bf-36eeef25f735.jpeg)
![截图未命名0817193308](https://user-images.githubusercontent.com/61811583/129719019-c6d3e3f1-840d-49a5-8238-6a86a1259149.jpeg)
The lowest note of NGP should be G2 or lower, but after conversion to AY8910, the lowest note is only A2

**Comment by okaxaki** [comment]
I have two questions.
- Does this issue only occur on NGP conversion? Is it okay for other SN76489 vgms? 
- Is the sound of the AY8910 detuned, or is it that the audible pitch is correct but the MDPlayer shows an incorrect pitch?

**Comment by okaxaki** [comment]
If your vgm's NGP clock is  307200Hz, please make sure pitch on the MDPlayer keyboard is correct.
If MDPlayer assumes 3579545Hz clock,  MDPlayer shows 2 lower semi-tones for a 307200Hz vgm.

**Comment by denjhang** [comment]
I can make sure that the pitch displayed by MDPlayer is correct. Because I have reported a problem in the pitch calculation of SN76489 with a frequency other than 3,575,545 hz a long time ago https://github.com/kuma4649/MDPlayer/issues/124. This problem was fixed soon, which means that currently MDPlayer considers the difference of the clock when calculating the pitch.

**Comment by denjhang** [comment]
Currently AY8910 sound is detuned. In NGP, all sounds lower than A2 become A2 of AY8910.

**Comment by denjhang** [comment]
[ngp-sonic.zip](https://github.com/digital-sound-antiques/vgm-conv/files/7001921/ngp-sonic.zip)

**Comment by denjhang** [comment]
I found that not only the NGP, but also the pitch conversion error for the SN76489 that does not use the 3559545hz clock.

**Comment by denjhang** [comment]
[lickspat.zip](https://github.com/digital-sound-antiques/vgm-conv/files/7002851/lickspat.zip)

**Comment by okaxaki** [comment]
Okay I got the point. Current vgm-conv internally convert SN76489's clock to 3579545Hz. At this clock, the note's lower limit is A2. I'll fix this problem but it will take some time.

Workaround: it is possible to get a correct conversion result with `-c` option that specifes the destination clock as the following.

```sh
vgm-conv -f sn76489 -t ay8910 -c 1536000 -o 1536000.vgm input.vgm
vgm-conv -c 1789773 -o output.vgm 1536000.vgm
```

**Comment by okaxaki** [comment]
Fixed at 0.8.21
